setup.py

```diff
@@ -240,8 +240,9 @@ def get_exts_for(name):
         "funcsigs>=1.0.0; python_version=='2.7'",
         "typing; python_version<'3.5'",
         "packaging>=17.1",
+        "protobuf>=3; python_version>='3.7'",
+        "protobuf>=3,<4.0; python_version<'3.7'",
```

Comment:
can we take a similar approach to the `bytecode` dependencies to tidy this up a bit?

Reply:
we can, and should do that in a follow-up PR